src/main/java/net/runelite/launcher/Launcher.java

```diff
@@ -148,115 +149,46 @@ public static void main(String[] args)
 		catch (IOException ex)
 		{
 			log.error("error fetching bootstrap", ex);
+			frame.setVisible(false);

```

Comment:
Just for the sake of less duplicated code, maybe this could be some panic() method or something